src/content/self-hosted/install/releases.mdx

```diff
@@ -8,11 +8,41 @@ id: releases
 ## 1.7.0
 TBD
 
+### Removal Notice
+- Kubernetes [1.21](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.21.md) is no longer supported.
+
 ### Features
 
 - Support for the annotation [dev.okteto.com/endpoints](cloud/ssl.mdx#customizing-the-endpoints-shown-in-the-okteto-ui) to customize the endpoints shown in the Okteto UI.
 - Support for configuring the [service account](self-hosted/administration/configuration.mdx#buildkit) of buildkit.
 - Support for [token-based authentication](self-hosted/administration/configuration.mdx#token) to simplify the installation of Okteto Self-Hosted.
+- [Personal Access Tokens](cloud/personal-access-tokens.mdx) now have expiration date. They expire after 180 days of their creation. For existing tokens, the expiration date is set to 180 days from when your instance is upgraded to this version
+- Update the Okteto CLI version to [2.14.1](https://github.com/okteto/okteto/releases/tag/2.14.1)
+- Allow to delete external resources individually from the UI
+- Default installation of Okteto doesn't require an external storage for the registry configured. It uses filesystem storage by default
+- Default installation of Okteto doesn't require a PVC for [BuildKit](self-hosted/administration/configuration.mdx#token). It uses [filesystem storage](self-hosted/administration/configuration.mdx#filesystem) by default
+
+### Improvements
+
+- When `buildkit.ingress.enabled` is true, BuildKit is exposed in the `buildkit.service.port` instead of port 1234. This makes easier to use other ingress controller.
+- Optimized redeploy dialog experience. Infomation loads faster in the dialog, specially when the source is GitHub
+- Improved confirmation dialogs messages
+- When your Okteto instance doesn't have an auth provider configured, the login screen shows a message with a link to the documentation about how to configure it
+- Upgrade ingress-nginx to [4.3.0](https://github.com/kubernetes/ingress-nginx/releases/tag/helm-chart-4.3.0)
+- Pod Security Policies are not installed if they are not available in the cluster
+- Moved `cloud.provider` helm chart value to [`registry.storage.provider`](self-hosted/administration/configuration.mdx#storage). We keep backward compatibility with values configured under `cloud.provider`
```

Comment:
Can we mention when do we plan to deprecate the old secret name? 

Also, should this be in a separate part since we want users to take action a soon as possible? Maybe a deprecation notice?

Reply:
> Can we mention when do we plan to deprecate the old secret name?

Not sure we have plans to completely remove this. Probably in chart `2.0` but I'll sync with product about it

> Also, should this be in a separate part since we want users to take action a soon as possible? Maybe a deprecation notice?

I had doubts about this. I don't know how emphasize we want to make on this migration for users. Helm output will let you know about the new keys but it's probably a good idea to mention it also here

Reply:
Having two different ways to accomplish the same thing makes it harder for our users to understand and for us to support them. If we don't have a clear plan to remove the old one, we should stick with the old one and not advertise the new one.  

I assume that we did the new one for a good reason, so I think the easier way is to have a clear deprecation path, and be transparent about it with our users.

Reply:
Added deprecation notice in this commit: https://github.com/okteto/docs/pull/365/commits/c8907ee6a9d2e0880686a4217cc90241d641011a